### ISSUE TITLE
Intellisense in VSCode Python

### ISSUE THREAD

**Issue by Raven888888** [pythonnet/pythonnet#2139]
### Environment

-   Pythonnet version: 3.0.1
-   Python version: 3.10
-   Operating System: Windows
-   .NET Runtime: 6.0

### Details

How do I enable intellisense and type hinting in vscode when using c# dll?

![image](https://user-images.githubusercontent.com/58241952/227986788-39c33036-38e5-4e4b-b9dc-c876e971009d.png)

**Comment by filmor** [comment]
That is completely out of scope for this library.

**Comment by Raven888888** [comment]
I agree, just curious what you guys using to make life easier, if not intellisense? Anyone has any better tool or workaround for vscode?
